.golangci.yml

```diff
@@ -245,6 +245,11 @@ linters:
         linters:
           - forbidigo
 
+        # FIXME(dmcgowan): ignoring while transitioning to containerd/errdefs
```

Comment:
Do you have a branch that updates the remaining uses? I was working on https://github.com/docker/cli/pull/6074, and while looking for uses, there were quite some remaining in the vendored code.

Also opened;

- https://github.com/moby/moby/pull/50012

Reply:
I had some branches but they got very out of date. In 29, will make that decision whether to pull it internal or update it completely. The marking as deprecated is for external importers mostly